bodhi/tests/server/__init__.py

```diff
@@ -4,23 +4,8 @@
 import sqlalchemy
 
 from bodhi.server.models import (
-    Base,
-    Bug,
-    Build,
-    BuildrootOverride,
-    Comment,
-    CVE,
-    Group,
-    Package,
-    Release,
-    ReleaseState,
-    Update,
-    UpdateType,
-    User,
-    UpdateStatus,
-    UpdateRequest,
-    TestCase,
-)
+    Bug, Build, BuildrootOverride, Comment,
+    CVE, Group, Package, Release, ReleaseState, Update, UpdateRequest, UpdateType, User, TestCase)
```

Comment:
Any reason for this split? Why not putting CVE, Group and Package in the first line?